src/main/resources/benchmarks/tpch/dialect-sqlserver.xml

```diff
@@ -0,0 +1,547 @@
+<?xml version="1.0"?>
+<dialects>
+    <dialect type="SQLSERVER">
+        <procedure name="Q1">
+            <statement name="query_stmt">
+                select
```

Comment:
Can you pretty-print these queries?

Reply:
Done. Please let me know if I followed the right convention (looked at a few other files).